intel_extension_for_transformers/llm/runtime/graph/scripts/convert_falcon.py

```diff
@@ -94,6 +94,16 @@ def main(args_in: Optional[List[str]] = None) -> None:
     fout.write(struct.pack("i", 0))
     fout.write(struct.pack("i", 0))  # word_embed_proj_dim (for opt)
     fout.write(struct.pack("i", 0))  # do_layer_norm_before (for opt)
+
+    fout.write(struct.pack("i", 0))
+    fout.write(struct.pack("i", 0))
+    fout.write(struct.pack("i", 0))
```

Comment:
please also add change to convert_bloom.py

Reply:
Done